dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1475,6 +1475,11 @@ case "$ACTION" in
       exit $?
     fi
 
+    if [ ! -d "${SERVER_OUTPUT_DIR}" ]; then
```

Comment:
Can we only do this `if [ ! -d "${SERVER_OUTPUT_DIR}" ] && [ ${ACTION} == "checkpoint" ]; then` instead?

That way we only change the case where a `checkpoint` is being done.